docs-source/content/userguide/managing-domains/model-in-image/reusing-rcu.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Reusing an RCU database"
+date = 2020-03-11T16:45:16-05:00
+weight = 70
+pre = "<b> </b>"
+description = "Reusing an RCU database between deployments of a Model in Image JRF domain."
++++
+
+#### Reusing an RCU database between JRF domain deployments
```

Comment:
there is no such thing as "an RCU database"

Reply:
What is the correct term?